.github/ISSUE_TEMPLATE/release-todo-issue-template.yml

```
@@ -55,9 +57,9 @@ body:
       
       | Release Types         | Label Used                                                                             |
       |-----------------------|----------------------------------------------------------------------------------------|
-      | Preview Release       | https://github.com/KinsonDigital/[TODO: add project name here]/labels/%F0%9F%9A%80Preview%20Release         |
-      | Production Release    | https://github.com/KinsonDigital/[TODO: add project name here]/labels/%F0%9F%9A%80Production%20Release      |
-      | Hot Fix Release       | https://github.com/KinsonDigital/[TODO: add project name here]/labels/%F0%9F%9A%80Hot%20Fix%20Release       |
+      | Preview Release       | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Preview%20Release         |
+      | Production Release    | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Production%20Release      |
+      | Hot Fix Release       | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Hot%20Fix%20Release       |
```

Please fix URL's.  They used to have the TODO instead of the project name, but when you replaced the entire template with Velaptor's, you forgot to fix the URL's to show the todo items.

```suggestion
      | Preview Release       | https://github.com/KinsonDigital/[TODO: add project name here]/labels/%F0%9F%9A%80Preview%20Release         |
      | Production Release    | https://github.com/KinsonDigital/[TODO: add project name here]/labels/%F0%9F%9A%80Production%20Release      |
      | Hot Fix Release       | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Hot%20Fix%20Release       |
```